### DISCUSSION TITLE
Would it be possible to add Brazilian Portuguese?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [StractOrg/stract#202] [Ideas]
I liked the project, but it's not yet usable outside of English, sites don't appear even when typing the entire url, what would be the problem?

**Comment by mikkeldenker** [comment]
Yea our index is still very small right now with only about 250 million pages, so the site probably hasn't been indexed yet. This is also why english queries tend to work best, but this should hopefully improve as the index grows

**Comment by Heus-Sueh** [comment]
> Yea our index is still very small right now with only about 250 million pages, so the site probably hasn't been indexed yet. This is also why english queries tend to work best, but this should hopefully improve as the index grows

Compared to Google, this is how many percent, can you measure this?

**Reply by mikkeldenker** [reply]
The most credible number I know about for [the size of Googles index](https://zyppy.com/seo/google-index-size/) is ~400 billion that came up during their antitrust trial. This would make our index approximately 0.06% the size of Googles, so we still have a long way to go but also a lot of great pages left to index
